src/main/java/uk/gov/digital/ho/hocs/document/DocumentConversionResource.java

```diff
@@ -66,6 +69,23 @@ public void convert(@RequestParam("file") MultipartFile file, HttpServletRespons
                 return;
             }
         }
+        if (msgDocumentConverter.isSupported(fileExtension)) {
```

Comment:
convert() method seem to be growing a lot, not a must have for this review but we should think about how to break up this method or make it shorter.

Reply:
this one is optional